lib/util.c

```diff
@@ -1460,6 +1460,47 @@ grn_inspect_limited(grn_ctx *ctx, grn_obj *buffer, grn_obj *obj)
   return buffer;
 }
 
+grn_obj *
+grn_inspect_key(grn_ctx *ctx, grn_obj *buffer, grn_obj *table, const void *key, unsigned int key_size)
+{
+  if (!buffer) {
+    buffer = grn_obj_open(ctx, GRN_BULK, 0, GRN_DB_TEXT);
+  }
+
+  if (!table) {
+    GRN_TEXT_PUTS(ctx, buffer, "(NULL)");
+    return buffer;
+  }
+
+  if (table->header.type == GRN_TABLE_NO_KEY) {
+    GRN_TEXT_PUTS(ctx, buffer, "#<key: (nil) table:#<");
```

Comment:
Ah, sorry. Could you remove `:` after `key`? `#<key (nil) ...>`.

Then we can merge this.

Reply:
I've fixed other same parts too.